tests/jballerina-debugger-integration-test/src/main/java/org/ballerinalang/debugger/test/utils/DebugTestRunner.java

```diff
@@ -83,7 +84,8 @@
     private static final int MAX_RETRY_COUNT = 3;
     private BMainInstance balClient = null;
     private Process debuggeeProcess;
-    private DebugHitListener listener;
+    private DebugHitListener hitListener;
+    private DebugOutputListener outputListener;
```

Comment:
`outputListener` is only used inside `waitForDebugOutput`. In that case, shall make this field to local variable in  `waitForDebugOutput` method?

Reply:
Changed in d318b5f